components/01-atoms/video/video.twig

```diff
@@ -0,0 +1,23 @@
+{#
+/**
+ * Available variables:
+ * - video_base_class - the base classname
+ * - video_modifiers - array of modifiers to add to the base classname
+ *   by default, videos are expected to be 16x9, include the modifier "full" to support a 4x3 video
+ * - video_blockname - blockname prepended to the base classname
+ * - video_content - the content of the video (typically an iframe)
+ *   the "|raw" filter is applied so that the iframe is rendered instead
+ *   of simply passing the iframe as plain text.
+ *
+ * Available blocks:
+ * - video_content - used to replace the content of the video with something other than the typical iframe
+ *   for example: to insert an html5 video component
```

Comment:
Please update the variables to use the double underscore method of separating the variable parts. So, change `video_content` to `video__content`.

Also, you can remove all of the BEM-related ones from the docs here. I figure that those are a given, and so to keep this section most helpful, we can remove the `base_class`, `modifiers`, and `blockname` variables from the docs... because they'd basically be the same thing in every file.